src/vivarium/interface/interactive.py

```diff
@@ -40,35 +37,23 @@ class InteractiveContext(SimulationContext):
     call to one of the helper methods provided in this module.
     """
 
-    def __init__(self, configuration: ConfigTree, components: List, plugin_manager: PluginManager = None):
-        super().__init__(configuration, components, plugin_manager)
-        self._initial_population = None
-
-    def setup(self):
-        """Setup the simulation and initialize its population."""
-        super().setup()
-        self._start_time = self.clock.time
-        self.initialize_simulants()
```

Comment:
I'm going to make the interactive context smart shortly so there will never be a need to call setup.